src/components/VersionBadge/style.css

```
@@ -0,0 +1,75 @@
+.celebration {
+  animation: shake-animation 1s ease 4;
+
+  & polygon {
+    fill: #2daec1;
+    animation:
+      spin 2s linear 2,
+      color-animation 1s ease 4;
+    transform-origin: center;
+    filter: drop-shadow(0 0 0.1rem gold);
+  }
+  &:hover {
+    animation: shake-animation 1s ease infinite;
+    & polygon {
+      animation:
+        spin 2s linear infinite,
+        color-animation 1s ease infinite;
+    }
+  }
+}
+@keyframes spin {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes color-animation {
+  0% {
+    fill: #2daec1;
+  }
+  10% {
+    fill: red;
+  }
+  40% {
+    fill: green;
+  }
+  60% {
+    fill: orange;
+  }
+  70% {
+    fill: #2daec1;
+  }
+  100% {
+    fill: #2daec1;
+  }
```

Alternative smoother colors:

```suggestion
  0% {
    fill: #b54e5d;
  }
  10% {
    fill: #b5813f;
  }
  40% {
    fill: #69a746;
  }
  60% {
    fill: #669b90;
  }
  70% {
    fill: #b54e5d;
  }
  100% {
    fill: #b54e5d;
  }
```